client/src/modules/Mentor/components/StudentsTable/StudentsTable.types.tsx

```diff
@@ -0,0 +1,8 @@
+import { MentorDashboardDto, ProfileCourseDto } from 'api';
+
+export type StudentsTableRow = MentorDashboardDto;
+
+export interface StudentsTableProps {
+  data: StudentsTableRow[];
+  course: ProfileCourseDto;
+}
```

Comment:
wondering why to creating separate file for types if they are using in single file

Reply:
I think, it was one of my experiments. I will move it to component for now. Thanks